src/userinfo/UserInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './UserInfo.scss';
+
+type User = {
+  name: string;
+  email: string;
+};
+
```

Comment:
move User type to the other types
```suggestion
type Props = {
  user: User;
};

```